.env.example

```
@@ -6,3 +6,12 @@ BASIC_AUTH_PASS=
 # Axiom logging (optional — falls back to console-only when missing)
 AXIOM_API_TOKEN=
 AXIOM_DATASET=ado-pulse
+
+# Nightly scheduler (comma-separated team names)
+SCHEDULED_TEAMS=
+ADO_ORG=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADO_ORG key should go before the SCHEDULED_TEAMS key

(UnorderedKey)